### PR TITLE
Add size checks before using JERRY_VLA in unittests

### PR DESCRIPTION
**Pull request by galpeter**

### CODE REVIEW

File: tests/unit-core/test-arraybuffer.c [hunk 1]
```diff
@@ -55,7 +55,8 @@ assert_handler (const jerry_call_info_t *call_info_p, /**< call information */
       && jerry_value_is_string (args_p[1]))
   {
     jerry_length_t utf8_sz = jerry_get_string_size (args_p[1]);
-    JERRY_VLA (char, string_from_utf8, utf8_sz);
+    TEST_ASSERT (utf8_sz <= 127);
```
Comment (rerobika) [inline tests/unit-core/test-arraybuffer.c:58]: Could you please specify that, where this 127 comes from?

Comment (galpeter) [reply tests/unit-core/test-arraybuffer.c:58]: I can add a comment "max assert message size" if it is ok for you.

Comment (rerobika) [reply tests/unit-core/test-arraybuffer.c:58]: Fine by me, significantly better than a magic constant.

**Review by rerobika — APPROVED** [review]

**Review by zherczeg — APPROVED** [review]
LGTM

**Pull request merged**